### PR TITLE
Feature/5 add environment variables for phpunit

### PR DESCRIPTION
**Pull request by finnlewis**

### CODE REVIEW

**Comment by finnlewis** [comment]
@ekes this look better? Seems to work for me!

File: README.md [hunk 1]
```diff
@@ -32,3 +32,42 @@ cd MY_PROJECT
...
+lando ssh
+phpunit web/modules/contrib/localgov_my_module
+```
+
+Tests can be filtered using the `--filter` option. To only run a specific test:
+
+```bash
+lando ssh
+phpunit --filter=myTestName
+```
```
Comment (stephen-cox) [inline README.md:73]: There's not need to`lando ssh` to run tests. Tests run fine with `lando phpunit`. Strangely, you do need to have SSHed into Lando to get test outputs saved.

**Review by stephen-cox — APPROVED** [review]
Added the environmental variables works. I'm not sure the docs on running tests in Lando are entirely correct.

File: README.md (hunk 1 above)
Comment (finnlewis) [reply README.md:73]: Yer, if you run `lando phpunit`, you don't see the links to the html output. 

So I'd recommend lando ssh, then running phpunit.

**Pull request merged**